packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -464,17 +464,28 @@ export const audiusBackend = ({
     const primary = creatorNodeGateways[0]
     const firstImageUrl = `${primary}${cid}`
 
-    if (disableImagePreload) {
-      return firstImageUrl
-    }
-
     if (primary) {
-      // Attempt to fetch/load the image using the first creator node gateway
-      const preloadedImageUrl = await preloadImage(firstImageUrl)
+      if (imagePreloader) {
+        try {
+          const preloaded = await imagePreloader(firstImageUrl)
```

Comment:
so this preloads the image which then get's used by the Image component when it's paired with a url that matches what's been preloaded?

Reply:
This also prevents a double fetch since we dont go through the cidCache + preloadImage section, and then the Image itself?

Reply:
^ yep that's exactly right.

this is all overly complex & there are demons everywhere

Reply:
well i started looking at this pr really not understanding the state of our images, so this was v helpful